students/hyeonjin/users/views.py

```diff
@@ -56,11 +55,16 @@ def post(self, request):
         try:
             email    = data['email']
             password = data['password']
-    
-            if not User.objects.filter(email=email, password=password).exists():
+
+            user = User.objects.get(email=email)
+            if not bcrypt.checkpw(password.encode('utf-8'), user.password.encode('utf-8')):
                 return JsonResponse({"message":"INVALID_USER"}, status=401)
-            return JsonResponse({"message":"SUCCESS"}, status=200)    
 
+            encoded_jwt = jwt.encode({'user_id': user.id}, SECRET_KEY, ALGORITHM)
```

Comment:
```suggestion
            encoded_jwt = jwt.encode({'user_id': user.id}, settings.SECRET_KEY, settings.ALGORITHM)
```
위의 리뷰 내용에 따라 다음과 같이 수정 할 수 있습니다.